index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header class="toolbar">
+        <a href="#" id="logo">My Logo</a>
+        <nav id="menu">
+            <a href="#"><img src="img/menu-icon.png" alt="menu icon"></a>
+        </nav>
+    </header>
+
+    <section>
+        <div class="headline">
+            <h1>HELLO ! THERE. I'm Antoine.</h1>
+            <h3 class="job-title">I'm a Software Developer</h3>
+            <p class="info">
+                I can help you build a product, feature or website. Look through some of my
+                work and experience! If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.
+        </p>
+            <ul class="socialmedia">
+                <li>
+                     <a href="https://github.com/ANTOINE1128"><img src="img/github-icon.png" alt="Github"></a>
+                </li>
+             <a href="https://twitter.com/TonyMakdessy"><img src="img/tweeter-icon.png" alt="Twitter"></a>
+                </li>
+                <a href="https://www.linkedin.com/in/antoine-makdessy/"><img src="img/linkden.png" alt="linkedin"></a>
+                </li>
+            </ul>
```

Comment:
- [ ] Kindly 😃 try to fix the li tags at the project here when I see the code I see that some li tag does not have a closing tag and some does not have an open tag and there is a conflict at the li at the ul tag, so please try to fix that for the project code that's for writing the code better without any bugs😊

| Your code| 
| --- |
|     ![image](https://user-images.githubusercontent.com/83360636/227562252-82e3b6a8-7ae9-46e0-a51f-ad821cf2b67d.png) |